aws/elasticache/redis/main.tf

```diff
@@ -0,0 +1,27 @@
+/*
+ * Create Elasticache subnet group
+ */
+resource "aws_elasticache_subnet_group" "redis_subnet_group" {
+  name       = var.subnet_group_name
+  subnet_ids = var.subnet_ids
+}
+
+/*
+ * Create Cluster
+ */
+resource "aws_elasticache_cluster" "redis" {
+  cluster_id           = replace(var.cluster_id, "/(.{0,20})(.*)/", "$1")
```

Comment:
Maybe add a comment to describe what's happening here.

Reply:
Heh, good question. I just copied it from the `memcache` module. I'll see if I can figure it out and write a comment.

Reply:
I guess it's pretty simple...just takes the first 20 characters.